### ISSUE TITLE
Enable MediaKeys pass through

### ISSUE THREAD

**Issue by stdedos** [slgobinath/SafeEyes#213]
Hi there,

Although mentioned here https://github.com/slgobinath/SafeEyes/issues/103 (and closed), Media Keys were not passed-through. Is that intentional?

Regardless, would you consider adding that?

**Comment by slgobinath** [comment]
Not intentional but this is a feature I also looking for :-)
Earlier I found it something hard to achieve according to the current implementation but will consider again.

**Comment by stdedos** [comment]
Isn't there a
```
'catch' keystroke, if X in []: ; send_key_globally(x)
```
?

(really rough lines from python/curses and some windows AHK scripting)

**Comment by slgobinath** [comment]
I don't know whether there is a `send_key_globally` or similar function.
FYI: This is how the Skip and Postpone shortcuts are handled by Safe Eyes: https://github.com/slgobinath/SafeEyes/blob/master/safeeyes/BreakScreen.py#L229

**Comment by stdedos** [comment]
Not python code per se (quick search reveals that it is kinda-of problematic in Unix), but you could execute `xdotool` https://askubuntu.com/questions/218496/sending-keyboard-commands-to-ubuntu-through-python-remote-for-my-blackberry

**Comment by r2rstep** [comment]
really looking forward to this improvement

**Comment by Vavooon** [comment]
Yes, sending some keys is hard, however it may be easier to use MPRIS interface.

**Comment by slgobinath** [comment]
Implemented in #295. Now there is a new plugin named: `Media Control` which supports pausing currently playing media players using dbus calls.

According to this implementation, if there is a media player playing while you take a break, you will get a pause button on the top right corner as shown in these screenshots. Currently, I have implemented only the Pause button assuming this is the most wanted option.

![screenshot from 2019-01-12 16-01-53](https://user-images.githubusercontent.com/4003168/51078612-ffd32580-1685-11e9-91a2-6af941028b62.png)
![screenshot from 2019-01-12 16-02-25](https://user-images.githubusercontent.com/4003168/51078613-02ce1600-1686-11e9-83f3-928dd8ca58c6.png)

**Comment by slgobinath** [comment]
@stdedos definitely +1 for your direction.
If that works, we can remove this plugin.
